.github/workflows/lean4checker_nightly.yml

```diff
@@ -0,0 +1,128 @@
+name: lean4checker on nighly-testing-YYYY-MM-DD Workflow
+
+## This workflow is identical to the `lean4checker.yml` workflow, except that it runs on the
+## most recent `nightly-testing-YYYY-MM-DD` tag instead of the `master` branch.
+## Please keep the workflow up to date with `lean4checker.yml`.
+
+on:
+  schedule:
+    - cron: '0 0 * * *'   # Runs at 00:00 UTC every day
+  workflow_dispatch:
+
+jobs:
+  check-lean4checker:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Fetch latest tags
+        run: |
+          git fetch --tags
+          # Find the most recent tag matching the pattern
+          LATEST_TAG=$(git tag | grep -E '^nightly-testing-[0-9]{4}-[0-9]{2}-[0-9]{2}$' | sort -r | head -n 1)
+          echo "LATEST_TAG=\"${LATEST_TAG}\"" >> "$GITHUB_ENV"
+
+      - name: Checkout latest tag branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ env.LATEST_TAG }}
+
+      - name: Cleanup
+        run: |
+          find . -name . -o -prune -exec rm -rf -- {} +
+          # Delete all but the 5 most recent toolchains
+          if cd ~/.elan/toolchains && find . -maxdepth 1 -type d ! -name . -print0 | xargs -0 ls -1td | grep -v 'nightly$' | grep -v 'stable$' | tail -n +6 | xargs -I {} sh -c 'echo {} && rm -rf "{}" && rm "../update-hashes/{}"'; then
+              : # Do nothing on success
+          else
+              : # Do nothing on failure, but suppress errors
+          fi
+
+      - name: Setup jq
+        uses: dcarbone/install-jq-action@v2.1.0
+
+      - name: Install elan
+        run: |
+          set -o pipefail
+          curl -sSfL https://github.com/leanprover/elan/releases/download/v3.1.1/elan-x86_64-unknown-linux-gnu.tar.gz | tar xz
+          ./elan-init -y --default-toolchain none
+          echo "$HOME/.elan/bin" >> "${GITHUB_PATH}"
+
+      - name: If using a lean-pr-release toolchain, uninstall
+        run: |
+          if [[ $(cat lean-toolchain) =~ ^leanprover/lean4-pr-releases:pr-release-[0-9]+$ ]]; then
+            printf 'Uninstalling transient toolchain %s\n' "$(cat lean-toolchain)"
+            elan toolchain uninstall "$(cat lean-toolchain)"
+          fi
+
+      - name: Print Lean and Lake versions
+        run: |
+          lean --version
+          lake --version
+
+      - name: Run lake exe cache get
+        run: |
+          lake exe cache get
+
+      - name: Prune ProofWidgets .lake
+        run: |
+          rm -rf .lake/packages/proofwidgets/.lake/build/lib
+          rm -rf .lake/packages/proofwidgets/.lake/build/ir
+          lake build ProofWidgets
+
```

Comment:
```suggestion
```
This was removed from the `bors.yml` / `build.yml` in #19729. I suspect we can remove it here (and in `lean4checker.yml`) too.